src/group_theory/group_action/basic.lean

```diff
@@ -3,6 +3,7 @@ Copyright (c) 2018 Chris Hughes. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Chris Hughes
 -/
+import algebra.hom.group_action
```

Comment:
I would expect the amount of imports here to decrease. I guess it's fine since there are only few transitive imports being added.

Reply:
Note that the goal here is to make fewer things import `group_theory/group_action/basic` (as opposed to importing fewer things), because it turns out that nothing in this file is basic; right now, even `data.module.basic` is transitively imported, despite `module` being a stronger typeclass than `mul_action`.

Reducing the number of things imported by `algebra.hom.group_action` is breaking various downstream files, which suggests that might have been a valuable thing for me to have pruned.

Reply:
Given that Eric has answered and Anne had already remarked "I guess it's fine" I'm marking this as resolved.